### PR TITLE
Remove static text for text "feature" (issue #2629)

### LINKED ISSUE #2629: Add a text "feature" instead of static text to /places page
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/views/notes/tools_places.html.erb](https://github.com/publiclab/plots2/blob/master/tools_places.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -20,15 +20,8 @@
 
 <div class="col-md-12">
 
-  <% if params[:action] == 'methods' %>
-    <div style="text-align:center;">
-      <h1 style="font-size:3.4em;margin-top:40px;"><%= raw t('notes.tools_places.tools_and_techniques') %></h1>
-      <h4 style="color:#888;"><%= raw t('notes.tools_places.subtitle') %></h4>
-      <p><i>Public Lab's community collaboratively develops many different techniques and methods for DIY environmental monitoring. Browse many of them here and learn how to get involved.</i></p>
-    </div>
-  <% else %>
-    <h2><%= t('notes.tools_places.public_lab_chapters') %> <small><%= raw t('notes.tools_places.start_a_chapter', :url1 => "/wiki/start-a-chapter") %></small></h2>
-  <% end %>
+  <%= feature('places') %>
+
   <hr />
   <% if params[:controller] == "notes" && (params[:action] == "popular" || params[:action] == "index" || params[:action] == "liked") %>
     <div class="navbar">
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by hyser521**
fixes #2629

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] ask `@publiclab/reviewers` for help, in a comment below
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [X] PR body includes `fixes #0000`-style reference to original issue #

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by hyser521** [comment]
@publiclab/reviewers, please review my changes.  I apologize for not committing from a feature branch- is there an easy way to edit this pull request so I can make the change appropriately?  Thank you for the help!

File: app/views/notes/tools_places.html.erb [hunk 1]
```diff
@@ -20,15 +20,7 @@
...
-  <% if params[:action] == 'methods' %>
-    <div style="text-align:center;">
-      <h1 style="font-size:3.4em;margin-top:40px;"><%= raw t('notes.tools_places.tools_and_techniques') %></h1>
-      <h4 style="color:#888;"><%= raw t('notes.tools_places.subtitle') %></h4>
-      <p><i>Public Lab's community collaboratively develops many different techniques and methods for DIY environmental monitoring. Browse many of them here and learn how to get involved.</i></p>
-    </div>
-  <% else %>
-    <h2><%= t('notes.tools_places.public_lab_chapters') %> <small><%= raw t('notes.tools_places.start_a_chapter', :url1 => "/wiki/start-a-chapter") %></small></h2>
-  <% end %>
+<%= feature('places') %>
```
Comment (seafr) [inline app/views/notes/tools_places.html.erb]: This line seems to require a little bit of indentation. It'd be great if you could fix that. And thank you for your first PR.

**Comment by seafr** [comment]
It's exactly how it should be.

This is now ready to be merged. Thank you for your contribution.

**Comment by jywarren** [comment]
This is fantastic! @steviepubliclab will be pleased. Thanks so much!!! 👍 👍 💪 🙌

**Comment by steviepubliclab** [comment]
Ohh this is great! Thanks so much! :)

**Pull request merged**